mlflow/sklearn/__init__.py

```diff
@@ -301,6 +303,8 @@ def save_model(
     # Save `requirements.txt`
     write_to(os.path.join(path, _REQUIREMENTS_FILE_NAME), "\n".join(pip_requirements))
 
+    _PythonEnv.current().to_yaml(os.path.join(path, _PYTHON_ENV_FILE_NAME))
```

Comment:
I added this line only in the sklearn flavor to save github actions runs. I'll add it in other flavors before the merge if everything looks good.